docs/consume-events.md

```diff
@@ -160,55 +160,89 @@ See for instance this Java code that consumes messages from a topic named “tes
+    kind: Component
     metadata:
-        name: event-name // <== This is what needs to be referenced
-        description: Event description
-        title: Event Name
-        // ...
-
-We use the `subscribesToEvent` property to generate [Backstage relations](https://backstage.io/docs/features/software-catalog/extending-the-model#adding-a-new-relation-type) between consumers and producers. These relations are displayed on Event Overview pages in a table. Without this property, there will be no visual representation of what systems or components are subscribing to a given event.
+        name: component-name
+        description: Component description.
+        title: Component Name
+        links:
+            - url: https://sample-slack-link.com
+            title: Event Consumer Slack Channel
+            - url: https://sample-link.com
+            title: Component Documentation
+    spec:
+        type: service
+        lifecycle: production
+        owner: owning-team
+        subscribesToEvent: [event-name, event-name-two]
+```
+
+Here is some additional information on these fields:
+
+* **apiVersion** [required]: This value must be set to `backstage.io/v1alpha1`.
+* **kind** [required]:  This value must be set to `Component`.
+* **metadata** [required]: A structure that contains information about the entity. The `metadata` structure includes the following properties.
+    * **name** [required]: A machine-readable name for the component. This value will be used in CODE VA urls, so it should be all lowercase and use hypens as separators.
+    * **description** [required]: A concise, high-level description of the event-consuming component.
+    * **title** [required]: A human-readable representation of the `name` to be used in CODE VA user interfaces.
+    * **links** [optional]: A list of links related to the component. Each link consists of a `url` and a `title`.
+        * **url** [required]: The external url that will be opened when the link is clicked.
+        * **title** [required]: Display text for the link.
+* **spec** [required]: A structure that contains information about the component. The `spec` structure includes the following properties.
+    * **type** [required]: The component type. Possible values include: `website`, `service`, `library`, etc.
+    * **lifecycle** [required]: The current development status for the component. Possible values include: `experimental`, `production`, `deprecated`, etc.
+    * **owner** [required]: The team that owns the event-consuming component. If this is set to the name of a GitHub team within the VA's GitHub organization, this field will link to a page with details about the team in CODE VA.
+    * **subscribesToEvent** [required]: An array of strings. Each string must match the `metadata.name` value of a producer's `catalog-info.yaml` file. This field is used to relate the component to the events that it consumes. All consumers of an event will be displayed on the Event Details page.
```

Comment:
Can we provide more context for the what the Event Details page is? or maybe link to the Events page?

Reply:
We could say something like
```
* **subscribesToEvent** [required]: An array of strings. Each string must match the `metadata.name` value of a producer's `catalog-info.yaml` file. This field is used to relate the system to the events that it consumes and to display the system on each related event's CODE VA catalog entry.
```

We could link to the DLA Event page, but that page doesn't currently have any consumers

Reply:
Yeah, I think that's better.